uninstaller/pre-delete.py

```diff
@@ -32,40 +32,126 @@
 
 
 def check_pfserver_status(service_name, namespace, port, user, password):
-    host = "%s.%s.svc.cluster.local" % (service_name, namespace)
-    print(host)
-    
+    print("check_pfserver_status service_name=[%s] namespace=[%s] port=[%s] user=[%s] password=[%s]" % (
+    service_name, namespace, port, user, password))
+    host = service_name
+
     client = paddleflow.Client(host, user, password, port)
     ret, response = client.login(user, password)
     if not ret:
         print(response)
-        err_msg = "client login failed, with host[%s], user[%s], password[%s], port[%s], response[%s]" % (host, user, password, port, response)
+        err_msg = "client login failed, with host[%s], user[%s], password[%s], port[%s], response[%s]" % (
```

Comment:
It might be better to change `print(response)` to `print(err_msg)` after line 43.